XenAdmin/Wizards/ImportWizard/ImportOptionsPage.cs

```diff
@@ -374,24 +396,13 @@ private void m_comboBoxISOLibraries_SelectedIndexChanged(object sender, EventArg
         private void m_radioButtonRunOSFixups_CheckedChanged(object sender, EventArgs e)
         {
             if (m_radioButtonRunOSFixups.Checked)
-            {
-                m_labelLocationFixupISO.Enabled = true;
-                m_comboBoxISOLibraries.Enabled = true;
                 ValidateSelection();
-            }
         }
 
         private void m_radioButtonDontRunOSFixups_CheckedChanged(object sender, EventArgs e)
         {
             if (m_radioButtonDontRunOSFixups.Checked)
-            {
-                m_labelLocationFixupISO.Enabled = false;
-                m_comboBoxISOLibraries.Enabled = false;
-                m_pictureBoxInfo.Visible = false;
-                m_labelFixupISOInfo.Visible = false;
-                m_buttonNextEnabled = true;
-                OnPageUpdated();
-            }
+                ValidateSelection();

```

Comment:
With this change, the Next button doesn't get enabled if DontRunOSFixup is checked